tests/test_confidence.py

```diff
@@ -72,7 +106,9 @@ def test_confidence_sigma_vs_prob(data, pars):
 
 def test_confidence_exceptions(data, pars):
     """Make sure the proper exceptions are raised when needed."""
-    minimizer = lmfit.Minimizer(residual, pars, calc_covar=False,
+    minimizer = lmfit.Minimizer(residual,
+                                pars,
+                                calc_covar=False,
```

Comment:
I would prefer not to change stylistic stuff (here and in the function definitions) without a clear reason. I've been trying to make the tests/code consistent with spaces and such and unless we agree on specific coding style; please leave follow surrounding code if possible

Reply:
The was accidentally done, since I did run a code format tool on the file. I would prefer to not change this back since I would end up fighting git for an hour.

Reply:
Fair enough, let's just leave it like this for now - we can always finale the coding-style once the whole test-suite is completed.